pkg/models/querybuilder_scene.go

```diff
@@ -10,33 +10,67 @@ import (
 	"github.com/stashapp/stash/pkg/database"
 )
 
-const scenesForPerformerQuery = `
-SELECT scenes.* FROM scenes
+var sceneMetadataColumns = []string{
+	"id",
+	"path",
+	"checksum",
+	"title",
+	"details",
+	"url",
+	"date",
+	"rating",
+	"size",
+	"duration",
+	"video_codec",
+	"audio_codec",
+	"format",
+	"width",
+	"height",
+	"framerate",
+	"bitrate",
+	"studio_id",
+	"created_at",
+	"updated_at",
+	"o_counter",
```

Comment:
"cover"
is missing

Reply:
I've reverted the change to `qb.All` which is what the export task uses, so it should get the cover again.

Reply:
confirmed , export is fine now.